.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,30 @@
+## Issue Type 
+
+(check one with "x")
+
+```
+[ ] bug report
+[ ] feature request
+```
+
+
+## Current Behavior
+
+
+## Expected Behavior
+
+
+## Steps to Reproduce
+
+
```

Comment:
Can you add:
### How can it be fixed?

Fill this in if you know how to fix it.